### ISSUE TITLE
[Idea]: Show and Tell: Make image alt text more clear

### ISSUE THREAD

**Issue by pjeweb** [alveusgg/alveusgg#566]
### Describe your idea / enhancement

I tried to make it clear the description is only show to screen readers, but we still get submissions that put text there that should be accessible to all users, not just as an alternative text describing the image.

![sat-alt](https://github.com/alveusgg/alveusgg/assets/684458/71222fa8-e3c9-4e1f-874c-464d19a05451)

I propose finding a better description/UI to make that clear. Maybe use "alternative text" or add "e.g. for blind users"?

### How would this idea / enhancement be beneficial?

Prevent incorrect entries, improve UX, make reviews easier.

**Comment by pjeweb** [comment]
Maybe the caption fields needs to be bigger?

(PS: I assume AI generated alt text services are still not super feasible, right?)

**Comment by MattIPv4** [comment]
Maybe `Visual Description: Text accessible to visually impaired users to describe the image`?
